src/App.tsx

```diff
@@ -1,6 +1,6 @@
 import type { ColorScheme } from "@mantine/core";
 import { AppShell, ColorSchemeProvider, MantineProvider } from "@mantine/core";
-import { Notifications } from "@mantine/notifications";
+import { NotificationsProvider } from "@mantine/notifications";
```

Comment:
This will make the build to fail. Mantine v6 does not have `NotificationsProvider` anymore.

Reply:
got it will make a new image and then push my pr will fix my issues ig